### PR TITLE
chore: adds user survey link in README and issue close workflow

### PR DESCRIPTION
**Pull request by alessbell**
Adds a link to our new user survey in the README as well as a workflow to add a comment after an issue is closed as completed.

### CODE REVIEW

**Comment by alessbell** [comment]
@calvincestari I requested your review as well since I wanted to confirm those links look good - I just swapped `Apollo+Client` for `Apollo+iOS`. Thanks!

File: .github/workflows/issue-close-user-survey.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  user-survey-comment:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    if: github.repository == 'apollographql/apollo-client'
```
Comment (calvincestari) [inline .github/workflows/issue-close-user-survey.yml:12]: What other repos would this run in; Is this a fork consideration?

**Review by calvincestari — APPROVED** [review]
Links are good - LGTM!

**Review by jerelmiller — APPROVED** [review]
🎉 Looking forward to using this!

File: .github/workflows/issue-close-user-survey.yml (hunk 1 above)
Comment (alessbell) [reply .github/workflows/issue-close-user-survey.yml:12]: Yes, just thinking about forks here

**Review by calvincestari — APPROVED** [review]

**Pull request merged**